paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsStateFactoryTest.kt

```diff
@@ -171,6 +174,78 @@ class PaymentOptionsStateFactoryTest {
         assertThat(options[2].isEnabledDuringEditing).isTrue()
     }
 
+    @Test
+    fun `State is correct when given non null defaultPaymentMethodId and it is selected`() {
+        val paymentMethods = PaymentMethodFixtures.createCards(3)
+
+        val selectedPaymentMethod = paymentMethods.random()
+        val defaultPaymentMethodId = selectedPaymentMethod.id
+
+        assert(defaultPaymentMethodId != null)
+
+        val state = PaymentOptionsStateFactory.create(
+            paymentMethods = paymentMethods,
+            showGooglePay = true,
+            showLink = true,
+            currentSelection = PaymentSelection.Saved(selectedPaymentMethod),
+            nameProvider = { it!!.resolvableString },
+            isCbcEligible = false,
+            canRemovePaymentMethods = true,
+            defaultPaymentMethodId = defaultPaymentMethodId
+        )
+
+        val options = state.items.filterIsInstance<PaymentOptionsItem.SavedPaymentMethod>()
+
+        val selectedPaymentOptionItem = options.find {
+            it.paymentMethod.id == defaultPaymentMethodId
+        }
+
+        assert(selectedPaymentOptionItem != null)
```

Comment:
for all the assertions in here, can you write them with `assertThat` statements -- e.g. `assertThat(value).isNotNull()`, `assertThat(...shouldShowDefaultBadge).isTrue()`, etc. 

It's easier to read that way